### ISSUE TITLE
feat(require-schema-suffix): consider adding an `allow` option to allow variable names to be used

### ISSUE THREAD

**Issue by marcalexiei** [marcalexiei/eslint-plugin-zod#72]
### 🧩 Description

Consider adding an `allow` option on `require-schema-suffix` to allow user to specify variable names.

At the beginning I would only consider string, but maybe this should also support regular expressions.

I'll leave the issue open to see if there are any opinion about this.

### ✒️ Additional context

- https://github.com/marcalexiei/eslint-plugin-zod-x/issues/71

----

```ts
const schema = z.object({
  code: z.string(),
  state: z.string(),
})
```

Should this one just be fine? 🤷🏼 it's a one-off schema. I'll rename it in my case

**Comment by MartinCura** [comment]
Thanks! (And quick work with those fixes.) My 2c: keep this as an issue and don't add the option unless more people ask for it. Cheers
